src/lessons/09_responsive_design/responsive-design.slides.html

```diff
@@ -0,0 +1,317 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>CSS and HTML + (incl. Flexbox) (part 2)</title>
+    <meta name="date" content="2021-04-13" />
```

Comment:
ohhh, just saw this 🤦🏼  sorry 

this made the slides not to sync in [live environment](https://redi-munich-frontend.github.io/fed-munich-2021-spring/)